recipe/meta.yaml

```diff
@@ -58,12 +57,14 @@ requirements:
     # avoid confusion from openblas bugs
     - libopenblas !=0.3.6  # [x86_64]
     - libopenblas >=0.3.18, !=0.3.20  # [arm64]
-    # CUDA 9.2 or later is required for CUDA support
-    - cudatoolkit >=9.2
+    # CUDA 10.2 or later is required for CUDA support
+    - cudatoolkit >=10.2
     # scipy 1.0 or later
     - scipy >=1.0
-    # numpy has a hard upper limit
-    - numpy >=1.18,<1.23
+    # NumPy has a hard upper limit.
+    # Also NumPy 1.22.0, 1.22.1, 1.22.2 are all broken for ufuncs
+    # xref: https://github.com/numba/numba/issues/7756
+    - numpy >=1.18,!=1.22.0,!=1.22.1,!=1.22.2,<1.23
```

Comment:
I feel like the lower bound here is wrong. It should depend on the build version which can be 1.19 or 1.21 depending on python 3.10

Reply:
Ah, nevermind, this is in the constraint, and the run pinning is unchanged. smart.